### PR TITLE
10% buff to Supermatter Energy and Radiation - Due to Station needing more power now

### PR DESCRIPTION
**Pull request by JamieD1**
With the map changes and the AI changes, more power is required and the starting setup is not quite handling it.

Small 10% buff to all supermatter power and radiation to compensate

:cl:  
tweak: Small 10% buff to all supermatter power and radiation to compensate
/:cl:

### CODE REVIEW

**Comment by Simplehorror** [comment]
STOP BUFFING THINGS LIKE THIS PLEASE

**Comment by Spacemanspark** [comment]
@Simplehorror While I might agree that power shouldn't be 'set up and forget', one also should consider that nobody is going to find it particularly interesting to play when half the station's roles are unable to do their jobs for a significant portion of time during a lot of rounds. Science/ medical/ botany/ kitchen/ bar/ etc.

Things sometimes need to be rebalanced when newer features are added. Sometimes these are nerfs, other times buffs.

**Comment by bruhlookatthisdood** [comment]
> @Simplehorror While I might agree that power shouldn't be 'set up and forget'

remove sm
replace with treadmills
make monkey army that works on treadmills

**Comment by Simplehorror** [comment]
i meant code wise

**Comment by ToasterBiome** [comment]
Don't know how I like this solution

File: code/modules/power/supermatter/supermatter.dm [hunk 1]
```diff
@@ -128,6 +128,9 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	var/temp_factor = 30
 	var/power = 0
 
+	/// Rad Modifier - Also Increases Power
```
Comment (ToasterBiome) [inline code/modules/power/supermatter/supermatter.dm]: no it doesnt

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
also simple is right

**Comment by Runian** [comment]
Go do solars. Solar plus SM is enough power for the station.

**Comment by Simplehorror** [comment]
To be quite dead simple, the AI should not be connected to the powernet but completely detached from the station and have solars like GaxStation.

File: code/modules/power/supermatter/supermatter.dm (hunk 1 above)
Comment (JamieD1) [reply code/modules/power/supermatter/supermatter.dm]: Yes it does? More rads = More Power? I tested this and it works as intended.

**Comment by JamieD1** [comment]
> Go do solars. Solar plus SM is enough power for the station.

Every other engine is more powerfull round start. This reasoning has no weight.

File: code/modules/power/supermatter/supermatter.dm (hunk 1 above)
Comment (ToasterBiome) [inline code/modules/power/supermatter/supermatter.dm]:
```suggestion
	/// Yogs - radiation modifier. After all power calculations, multiplies the intensity of the rad pulse by this value. Used for making engines more hugbox.
```

File: code/modules/power/supermatter/supermatter.dm [hunk 2]
```diff
@@ -496,7 +499,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 
 		if(prob(50))
 			//1 + (tritRad + pluoxDampen * bzDampen * o2Rad * plasmaRad / (10 - bzrads))
-			radiation_pulse(src, max(power * (1 + (power_transmission_bonus/(10-(bzcomp * BZ_RADIOACTIVITY_MODIFIER))))))
+			radiation_pulse(src, max((power * (1 + (power_transmission_bonus/(10-(bzcomp * BZ_RADIOACTIVITY_MODIFIER))))) * radmodifier))
```
Comment (ToasterBiome) [inline code/modules/power/supermatter/supermatter.dm]:
```suggestion
			radiation_pulse(src, max((power * (1 + (power_transmission_bonus/(10-(bzcomp * BZ_RADIOACTIVITY_MODIFIER))))) * radmodifier)) //Yogs addition, radmodifier
```

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

**Comment by Simplehorror** [comment]
This commit was created on GitHub.com and signed with GitHub’s verified signature.

**Comment by JamieD1** [comment]
> This commit was created on GitHub.com and signed with GitHub’s verified signature.

Why do you keep saying this?

**Pull request merged**